docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,89 @@ To process the received search command on the server side, follow these steps:
 > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
 
+## Updating and Retrieving Abilities
+To update the abilities on the server side, follow these steps:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+   
+2. Set the abilities using the corresponding 'mc_server_set_XXX_ability()', or 'mc_server_set_ability_support()'.
+   The following examples demonstrate setting up various abilities:
+   
+   To update the playback ability, set the ability for each playback action using the 'mc_server_set_playback_ability()', and then update the ability using the 'mc_server_update_playback_ability()':
```

Comment:
To update the playback ability, set the ability for each playback action using 'mc_server_set_playback_ability()', and then update the ability using  'mc_server_update_playback_ability()':

Remove "the"

Reply:
Please change ' to `.

 To update the playback ability, set the ability for each playback action using the `mc_server_set_playback_ability()`, and then update the ability using the `mc_server_update_playback_ability()`:

Reply:
Done